Client/mods/deathmatch/logic/luadefs/CLuaGUIDefs.cpp

```diff
@@ -3887,6 +3892,71 @@ int CLuaGUIDefs::GUIComboBoxSetItemText(lua_State* luaVM)
     return 1;
 }
 
+int CLuaGUIDefs::GUIComboBoxGetItemCount(lua_State* luaVM)
+{
+    // int guiComboBoxGetItemCount( element comboBox )
+    CClientGUIElement* comboBox;
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadUserData<CGUIComboBox>(comboBox);
+
+    if (!argStream.HasErrors())
+    {
+        int items = CStaticFunctionDefinitions::GUIComboBoxGetItemCount(*comboBox);
+        lua_pushnumber(luaVM, items);
+        return 1;
+    }
+    else
+        m_pScriptDebugging->LogCustom(luaVM, argStream.GetFullErrorMessage());
+
+    // error: bad arguments
+    lua_pushboolean(luaVM, false);
+    return 1;
+}
+
+int CLuaGUIDefs::GUIComboBoxSetState(lua_State* luaVM)
+{
+    // int guiComboBoxSetState( element comboBox, int state)
+    CClientGUIElement* comboBox;
+    int state;
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadUserData<CGUIComboBox>(comboBox);
+    argStream.ReadNumber(state);
```

Comment:
I'd personally prefer an enum string here (and ofc return a string in `guiComboxGetState`). I'd like to hear other opinions though.

**Or, an even better solution - as you only distinguish two states - is to rename the function to `guiComboxIsOpen` and make it return a _boolean_.**